addons/website_slides/static/src/xml/website_slides_share.xml

```diff
@@ -8,42 +8,67 @@
 
     <t t-name="website.slide.share.socialmedia">
         <div class="row">
-            <div class="col-12 col-lg-6 mb-4">
-                <h5 class="mt-0 mb-2">Share on Social Networks</h5>
-                <div class="btn-group" role="group">
-                    <a t-attf-href="https://www.facebook.com/sharer/sharer.php?u=#{widget.slide.websiteShareUrl}" class="btn border bg-white o_wslides_js_social_share" social-key="facebook" aria-label="Share on Facebook" title="Share on Facebook"><i class="fa fa-facebook-square fa-fw"/></a>
-                    <a t-attf-href="https://twitter.com/intent/tweet?text=#{widget.slide.name}&amp;url=#{widget.slide.websiteShareUrl}" class="btn border bg-white o_wslides_js_social_share"  social-key="twitter" aria-label="Share on Twitter" title="Share on Twitter"><i class="fa fa-twitter fa-fw"/></a>
-                    <a t-attf-href="http://www.linkedin.com/sharing/share-offsite/?url=#{widget.slide.websiteShareUrl}" social-key="linkedin" class="btn border bg-white o_wslides_js_social_share" aria-label="Share on LinkedIn" title="Share on LinkedIn"><i class="fa fa-linkedin fa-fw"/></a>
-                </div>
-            </div>
-            <div class="col-12 col-lg-6">
+            <div class="col-12">
                 <h5 class="mt-0 mb-2">Share Link</h5>
                 <div class="input-group">
-                    <input type="text" class="form-control o_wslides_js_share_link" t-att-value="widget.slide.websiteShareUrl" readonly="readonly" onClick="this.select();" />
-                    <button class="btn btn-sm btn-primary o_clipboard_button" style="border-top-end-radius: 4px;border-bottom-end-radius: 4px;" >
-                        <span class="fa fa-clipboard"> Copy Link</span>
+                    <input type="text" class="form-control o_wslides_js_share_link text-center" t-att-value="widget.slide.websiteShareUrl" readonly="readonly" onClick="this.select();" />
+                    <button class="btn btn-sm btn-primary o_clipboard_button" style="border-top-right-radius: 4px;border-bottom-right-radius: 4px;" >
+                        <i class="fa fa-clipboard"/> Copy Link
                     </button>
                 </div>
             </div>
-            <div t-attf-class="col-12 col-lg-6">
+            <div class="col-12 mt-4">
```

Comment:
It seems like most of this commit is also related to improving the share feature in the website_slides module.

I would re-group all the changes related to sharing in a single commit and maybe make a separate commit for the miscellaneous improvements, such as the small change in "website_slides_forum" and the changes in the review labels.